src/types/Schedule.ts

```diff
@@ -1,5 +1,5 @@
 export interface Schedule {
-    dateOfWeek: string;
-    start: string;
-    end: string;
+    dateOfWeek: number;
+    start: number;
+    end: number;
```

Comment:
Same with clarifying what `start` and `end` mean? (ie the hour of the day in ET)